igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/client/model/BuildArtifact.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2014 Netflix, Inc.
+ * Copyright 2019 Pivotal, Inc.
```

Comment:
Do not change license headers on updates.

Reply:
👍